src/indexers/vault-definition-indexer.spec.ts

```diff
@@ -35,7 +35,7 @@ describe('vault-definition-indexer', () => {
         return { sett: <gqlGenT.SettQuery['sett']>graphVault, __typename: 'Query' };
       });
 
-      await mockSupportedChains();
```

Comment:
why u have removed this one? looks like `setupMockChain` mocks only 1 chain, and that helper was mocking `sdk`  and providers for all available chains